main.go

```diff
@@ -557,6 +557,11 @@ func checkOneInitialCollectionStatus(ctx context.Context, server *state.Server,
 	if err != nil {
 		return err
 	}
+	autodetectedDataDir := postgres.GetDataDirectory(server, settings)
```

Comment:
Maybe `dataDirectorySetting`? (not sure if we're doing much auto-detecting here specifically besides getting that setting)